lib/userlike/index.js

```diff
@@ -55,6 +63,34 @@ Userlike.prototype.initialize = function(page){
  *
  * @return {Boolean}
  */
+
 Userlike.prototype.loaded = function(){
   return !! (window.userlikeConfig && window.userlikeData);
 };
+
+/**
+ * Listen for chat events.
+ */
+
+Userlike.prototype.attachListeners = function(){
+  var self = this;
+  window.userlikeTrackingEvent = function(event_name, global_ctx, session_ctx){
+    if (event_name === 'chat_started') {
+      self.analytics.track('Live Chat Conversation Started',
+        { agentId: session_ctx.operator_id, agentName: session_ctx.operator_name },
+        { context: { integration: integration }});
+    } else if (event_name === 'message_operator_terminating') {
+      self.analytics.track('Live Chat Message Sent',
+        { agentId: session_ctx.operator_id, agentName: session_ctx.operator_name },
+        { context: { integration: integration }}); // message not available
```

Comment:
Can we change all of these "message not available"s into a todo note so we know to revisit it later?

```js
// TODO: As of 4/17/2015, Userlike doesn't give access to the message body in events. Revisit this/send it when they do
```